src/api/registrationApi.ts

```diff
@@ -30,7 +35,7 @@ export const unpublishRegistration = async (
   });
 
 export const getRegistrationByDoi = async (doiUrl: string) => {
-  const getRegistrationByDoiResponse = await authenticatedApiRequest2<Doi>({
+  const getRegistrationByDoiResponse = await authenticatedApiRequest2<DoiPreview>({
     url: `${PublicationsApiPath.DoiLookup}/`,
```

Comment:
(ikke noe du har gjort, men bare irriterende å se 😅 )
```suggestion
    url: PublicationsApiPath.DoiLookup,
```